dev/com.ibm.ws.jaxws.2.3.ejb/bnd.bnd

```diff
@@ -0,0 +1,98 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+          
+bVersion=1.0
+
+Bundle-Name: JAX-WS EJB
+Bundle-SymbolicName: com.ibm.ws.jaxws.2.3.ejb
+Bundle-Description: IBM JAX-WS EJB support; version=${bVersion}
+
+cxfVersion=3.2.6
```

Comment:
I think you should be able to get rid of this variable since I don't see it being used anywhere (I might have missed it though)